### PR TITLE
Mark no longer flakey tests as bringup:false 

### PR DESCRIPTION
**Pull request by reidbaker**
This is a result of auditing the ci.yaml tests marked as bringup true and determining which ones were still flakey. 
Each commit markes one tests as non flakey along with a link of proof of the lack of flakes. 

- Mark new_gallery_impeller__transition_perf as not flakey, #124699 closed, not recent flakes
- No failures in 500 runs for Staging_build_linux analyze https://ci.chromium.org/p/flutter/builders/staging/Staging_build_linux%20analyze\?cursor\=id%3E8782771663962116337\&limit\=200
- No flakes in 600 builds https://ci.chromium.org/p/flutter/builders/staging/Linux%20flutter_packaging_test\?cursor\=id%3E8787892377160059793\&limit\=200
- No flakes in 600 builds https://ci.chromium.org/p/flutter/builders/staging/Mac%20flutter_packaging_test\?cursor\=id%3E8787870631931126673\&limit\=200
- No flakes in 600 builds https://ci.chromium.org/p/flutter/builders/staging/Mac_arm64%20flutter_packaging_test\?cursor\=id%3E8787809799161254193\&limit\=200

#124699 is the only closed bug that was linked for a test that is no longer flakey. Closed status helps the case that flakes are no longer an issue. 

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

### CODE REVIEW

**Review by godofredoc — APPROVED** [review]

File: .ci.yaml [hunk 1]
```diff
@@ -4808,7 +4808,7 @@ targets:
     recipe: packaging/packaging
     timeout: 60
     scheduler: release
-    bringup: true
+    bringup: false
```
Comment (keyonghan) [inline .ci.yaml:4811]: This target has been timing out frequently: https://ci.chromium.org/p/flutter/builders/prod/Linux%20flutter_packaging_test?limit=50, we do not want to simply flip it back.

File: .ci.yaml [hunk 2]
```diff
@@ -4821,7 +4821,7 @@ targets:
...
```
Comment (keyonghan) [inline .ci.yaml:4824]: Instead, we can remove the `bringup` entry.

File: .ci.yaml [hunk 3]
```diff
@@ -4837,7 +4837,7 @@ targets:
...
```
Comment (keyonghan) [inline .ci.yaml:4840]: Ditto.

File: .ci.yaml [hunk 4]
```diff
@@ -2456,7 +2456,7 @@ targets:
 
   - name: Staging_build_linux analyze
     presubmit: false
-    bringup: true
+    bringup: false
```
Comment (keyonghan) [inline .ci.yaml:2459]: Instead, we may want to remove this target: https://github.com/flutter/flutter/issues/126278

File: .ci.yaml (hunk 1 above)
Comment (reidbaker) [reply .ci.yaml:4811]: Set to true and filed https://github.com/flutter/flutter/issues/126286

File: .ci.yaml (hunk 4 above)
Comment (keyonghan) [reply .ci.yaml]: Thanks for the link. IIRC, these `Staging_build` targets were created for experiment purpose, not expected to affect the tree. @yusuf-goog Has that experiment done? If yes, we can remove these targets.

File: .ci.yaml (hunk 1 above)
Comment (keyonghan) [reply .ci.yaml]: Yes, we should treat them the same, whenever the test becomes flaky and caused tree red.

File: .ci.yaml (hunk 4 above)
Comment (reidbaker) [reply .ci.yaml]: https://ci.chromium.org/p/flutter/builders/prod/Linux%20analyze?limit=200

**Comment by reidbaker** [comment]
Holding autosubmit on @keyonghan's lgtm

File: .ci.yaml (hunk 4 above)
Comment (keyonghan) [reply .ci.yaml]: The `Staging_build_linux analyze` is same as `Linux analyze`.  We do not want to run two same tests in prod CI.
Before confirmed by Yusuf whether we want to delete it, I would suggest keep the `bringup: true` to unblock here.

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**